StaticPagesPlugin.php

```diff
@@ -200,7 +200,7 @@ public function getActions($request, $actionArgs)
                         null,
                         'management',
                         'settings',
-                        'website',
+                        ['website'],
```

Comment:
There's a bug when managing a static page item. It couldn't build the URL for the link action for the item because it expects the `$path` param to be an array.